GVFS/GVFS.Platform.Windows/WindowsPlatform.cs

```diff
@@ -23,7 +24,7 @@ public partial class WindowsPlatform : GVFSPlatform
         private const string BuildLabExRegistryValue = "BuildLabEx";
 
         public WindowsPlatform()
-            : base(executableExtension: ".exe")
+            : base(executableExtension: ".exe", installerExtension: ".exe")

```

Comment:
Shouldn't this be using the `InstallerExtension` const?

Reply:
@alameenshah  please put this on your cleanup list

Reply:
Added to the cleanup tasks.